perun-core/src/test/java/cz/metacentrum/perun/core/entry/MembersManagerEntryIntegrationTest.java

```diff
@@ -1714,6 +1717,32 @@ public void findOnlySponsoredMembers() throws Exception {
 		assertTrue(members.size() == 0);
 	}
 
+	@Test
+	public void removeLastSponsorWithoutExpiration() throws Exception {
+		System.out.println(CLASS_NAME + "removeLastSponsorWithoutExpiration");
+
+		//Set up sponsor
+		Member sponsorMember = setUpSponsor(createdVo);
+		User sponsorUser = perun.getUsersManagerBl().getUserByMember(sess, sponsorMember);
+		AuthzResolverBlImpl.setRole(sess, sponsorUser, createdVo, Role.SPONSOR);
+		//Set up expiration rule
+		Map<String, String> rulesMap = new LinkedHashMap<>();
+		rulesMap.put("doNotExpireSponsoredMembers", "true");
```

Comment:
Even there you probably can use the static variable from the module.

Reply:
Fixed.